lib/zip/entry.rb

```diff
@@ -83,7 +83,7 @@ def time
       end
     end
 
-    alias :mtime :time
+    alias_method :mtime, :time
```

Comment:
There is no sense to use alias_method because we don't need chains. So better use alias instead of alias_method